docs/ucx/docs/installation/index.mdx

```diff
@@ -0,0 +1,234 @@
+
+```commandline
+databricks labs install ucx
+```
+
+You'll be prompted to select a [configuration profile](https://docs.databricks.com/en/dev-tools/auth.html#databricks-client-unified-authentication) created by `databricks auth login` command.
+
+Once you install, proceed to the [assessment workflow](/docs/reference/workflows#assessment-workflow) to ensure the compatibility of your workspace with UCX.
+
+The `WorkspaceInstaller` class is used to create a new configuration for Unity Catalog migration in a Databricks workspace.
+It guides the user through a series of prompts to gather necessary information, such as selecting an inventory database, choosing
+a PRO or SERVERLESS SQL warehouse, specifying a log level and number of threads, and setting up an external Hive Metastore if necessary.
+Upon the first installation, you're prompted for a workspace local [group migration strategy](/docs/reference/group_name_conflict).
+Based on user input, the class creates a new cluster policy with the specified configuration. The user can review and confirm the configuration,
+which is saved to the workspace and can be opened in a web browser.
+
+The [`WorkspaceInstallation`](https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/install.py) manages the installation and uninstallation of UCX in a workspace. It handles
+the configuration and exception management during the installation process. The installation process creates dashboards, databases, and jobs.
+It also includes the creation of a database with given configuration and the deployment of workflows with specific settings. The installation
+process can handle exceptions and infer errors from job runs and task runs. The workspace installation uploads wheels, creates cluster policies,
+and wheel runners to the workspace. It can also handle the creation of job tasks for a given task, such as job dashboard tasks, job notebook tasks,
+and job wheel tasks. The class handles the installation of UCX, including configuring the workspace, installing necessary libraries, and verifying
+the installation, making it easier for users to migrate their workspaces to UCX.
+At the end of the installation, the user will be prompted if the current installation needs to join an existing collection (create new collection if none present).
+For large organization with many workspaces, grouping workspaces into collection helps in managing UCX migration at collection level (instead of workspaces level)
+User should be an account admin to be able to join a collection.
+
+After this, UCX will be installed locally and a number of assets will be deployed in the selected workspace.
+These assets are available under the installation folder, i.e. `/Applications/ucx` is the default installation folder. Please check [here](#force-install-over-existing-ucx) for more details.
+
+You can also install a specific version by specifying it like `@v0.13.2` - `databricks labs install ucx@v0.13.2`.
```

Comment:
Make this a subsection: Installing specific release

Reply:
Done 👍